lib/checks/keyboard/landmark-is-top-level.js

```diff
@@ -0,0 +1,20 @@
+const landmarks = axe.commons.aria.getRolesByType('landmark');
+const sectioning = ['article', 'aside', 'main', 'navigation', 'section'];
+const nodeIsHeader = node.tagName.toLowerCase() === 'header' && node.getAttribute('role') !== 'banner';
+const nodeIsFooter = node.tagName.toLowerCase() === 'footer' && node.getAttribute('role') !== 'contentinfo';
+var parent = axe.commons.dom.getComposedParent(node);
+
+while (parent){
+	var role = parent.getAttribute('role');
+	if (!role && (parent.tagName.toLowerCase() !== 'form')){
+		role = axe.commons.aria.implicitRole(parent);
+	}
+  if (role && (nodeIsHeader || nodeIsFooter) && sectioning.includes(role)){
```

Comment:
The indentation here is a little funky. This is also super nitpicky, but we use a space before the opening curly brace in function declarations and conditional statements: `function() {` and `if (conditional) {`

Reply:
Update the code to remove extra spacing